core/services/pipeline/common_eth.go

```diff
@@ -90,12 +96,173 @@ func parseETHABIString(theABI []byte, isLog bool) (name string, args abi.Argumen
 		return "", nil, nil, errors.Errorf("bad ABI specification: %v", theABI)
 	}
 	name = string(bytes.TrimSpace(matches[0][1]))
-	if len(name) == 0 {
```

Comment:
This checks to make sure that the `abi` attribute contains the log name, but there was no functional value in doing so. Not married to removing it, though -- happy to add it back if people prefer